### PR TITLE
(58)(ui)create units loader collection

### PR DESCRIPTION
**Pull request by Ohuryayeva**

### CODE REVIEW

File: ui/Gruntfile.js [hunk 1]
```diff
@@ -33,6 +33,7 @@ module.exports = function(grunt) {
         'js/model/auth.js',
         'js/model/alert-message.js',
 		'js/collection/unit-collection.js',
```
Comment (Gorgonello) [inline ui/Gruntfile.js]: Please make sure that all is good with your indentation into gruntfile!

File: webservice/handlers/unit.js [hunk 2]
```diff
@@ -46,10 +47,12 @@ exports.getByRange = function(req, res) {
     var query={};
     query.time = {};
     if (req.query.startDate) {
-        query.time.$gte =  new Date(req.query.startDate).getTime();
+        //query.time.$gte =  new Date(req.query.startDate).getTime();
```
Comment (Gorgonello) [inline webservice/handlers/unit.js]: Please remove commented code

File: webservice/handlers/unit.js [hunk 3]
```diff
@@ -46,10 +47,12 @@ exports.getByRange = function(req, res) {
...
+        query.time.$gte =  req.query.startDate;
     }
     if (req.query.endDate) {
-        query.time.$lte = new Date(req.query.endDate).getTime();
+        //query.time.$lte = new Date(req.query.endDate).getTime();
```
Comment (Gorgonello) [inline webservice/handlers/unit.js]: The same stuff

File: ui/Gruntfile.js (hunk 1 above)
Comment (Ohuryayeva) [inline ui/Gruntfile.js]: In my gruntfile everything is fine, so i don't know how to fix it.

**Pull request merged**